pkg/operator/ceph/pool/controller_test.go

```diff
@@ -293,7 +293,7 @@ func TestCephBlockPoolController(t *testing.T) {
 
 	// Register operator types with the runtime scheme.
 	s := scheme.Scheme
```

Comment:
Add this 

```
	defer func() {
		s = runtime.NewScheme()
	}()
```

That will fix your test issue.

Reply:
Rather than re-creating here, I created a new Scheme at the beginning of the concerned Test,